src/avt/Expressions/CMFE/avtConnCMFEExpression.C

```diff
@@ -220,8 +224,9 @@ avtConnCMFEExpression::ExecuteTree(avtDataTree_p in1, avtDataTree_p in2,
             bool isCell =
                 in_ds2->GetPointData()->GetArray(invar.c_str()) == NULL &&
                 in_ds2->GetCellData()->GetArray(invar.c_str()) != NULL;
-            if ((orig1 == NULL && orig2 != NULL) || 
-                (orig1 != NULL && orig2 == NULL) && isCell)
+
+            if ( ( (orig1 == NULL && orig2 != NULL) || 
```

Comment:
I agree it could have been a bug pointed out by the warning. 

I reviewed the code and reworked the parens to match my understanding of what we intended.